common/src/main/java/com/wynntils/features/utilities/TranscribeMessagesFeature.java

```diff
@@ -144,9 +157,23 @@ private StyledText getStyledTextWithTranscription(
                         defaultColor);
             }
 
+            Style style;
+
+            if ((transcribeGavellian || transcribeWynnic) && !showTooltip.get()) {
+                style = Style.EMPTY.withHoverEvent(new HoverEvent(
+                        HoverEvent.Action.SHOW_TEXT,
+                        Component.translatable("feature.wynntils.transcribeMessages.transcribedFrom")));
+            } else if ((transcribeGavellian || transcribeWynnic) && showTooltip.get()) {
```

Comment:
Also, I think it is better to have this as a nested if: first check if we have any of the two transcriptions, and then decide the style  etc depending on the showTooltip config.